src/Transfer/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Include function for adding Charm++ modules
+include(charm)
+
+add_library(Worker
+            Worker.cpp
+            Controller.cpp)
```

Comment:
We could choose a better name instead of 'Worker' and 'Controller'. The former is lower level to the library, the latter is the interface to call its functionality. I propose renaming 'Controller' to 'Transfer' and 'Worker' to 'TransferDetail'.